### PR TITLE
Add Hero Homepage

### PR DESCRIPTION
**Pull request by matijs**
image

The `hero-homepage` is the full height of the viewport minus the height of the top navigation (`60px`).
Browsers that support WebP will use the WebP variant served by Contentful, Safari will use JPEG. Browsers without `<picture>` support default to the JPEG from the ``.
The `buildSrcSet` function needs some tweaking to output a fairly well distributed set of URLs and widths.

### CODE REVIEW

File: src/js/components/organisms/hero-homepage/helpers/build-srcset.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const SCREEN_WIDTHS: number[] = [640, 750, 828, 1125, 1242];
```
Comment (zwaardje) [inline src/js/components/organisms/hero-homepage/helpers/build-srcset.ts]: Are these not defined elsewhere?

File: src/js/components/organisms/hero-homepage/hero-homepage.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+$headerHeight: 60px;
```
Comment (zwaardje) [inline src/js/components/organisms/hero-homepage/hero-homepage.scss]: Used only once, no need to define the variable then.

File: src/js/components/organisms/hero-homepage/hero-homepage.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.hero-homepage__image {
+    height: 100%;
+    left: 0;
+    object-fit: cover;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    // pull the image behind the backdrop
```
Comment (zwaardje) [inline src/js/components/organisms/hero-homepage/hero-homepage.scss]: That's logical is it not?

File: src/js/components/organisms/hero-homepage/hero-homepage.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    name: 'hero-homepage',
+})
+export default class HeroHomepage extends Vue implements HeroHomepageProps {
+    @Prop({ type: String, required: true }) title!: HeroHomepageProps['title'];
+    @Prop({ type: String, required: false }) description?: HeroHomepageProps['description'];
+    @Prop({ type: Object, required: false }) image?: HeroHomepageProps['image'];
+    @Prop({ type: Object, required: false }) ctas?: HeroHomepageProps['ctas'];
+
+    get srcSet(): string {
+        const { url = '' } = this.image || {};
```
Comment (zwaardje) [inline src/js/components/organisms/hero-homepage/hero-homepage.ts]: Please convert to a better readable if statement. More letters are not always worse.

File: src/js/components/organisms/hero-homepage/hero-homepage.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        return url ? buildSrcSet({ url }) : '';
+    }
+
+    get srcSetWebP(): string {
```
Comment (zwaardje) [inline src/js/components/organisms/hero-homepage/hero-homepage.ts]: The same applies here.

File: src/js/components/organisms/hero-homepage/helpers/build-srcset.ts (hunk 1 above)
Comment (matijs) [reply src/js/components/organisms/hero-homepage/helpers/build-srcset.ts]: Not for now. We need to come up with some sensible values that work for this full width component.

For other components that only show images at for example 200px wide different values are needed.

File: src/js/components/organisms/hero-homepage/hero-homepage.scss (hunk 2 above)
Comment (matijs) [reply src/js/components/organisms/hero-homepage/hero-homepage.scss]: I'll add a comment below instead 👍

File: src/js/components/organisms/hero-homepage/hero-homepage.scss (hunk 3 above)
Comment (matijs) [reply src/js/components/organisms/hero-homepage/hero-homepage.scss]: Maybe not for everyone.

**Pull request merged**